google/cloud/spanner/row.h

```diff
@@ -165,7 +165,9 @@ class Row {
   friend bool operator==(Row const& a, Row const& b) {
     return a.values_ == b.values_;
   }
-  friend bool operator!=(Row const& a, Row const& b) { return !(a == b); }
+  friend bool operator!=(Row const& a, Row const& b) {
+    return !(a == b);
```

Comment:
No, I'm thinking 
```cc
return a.value_ != b.value_;
```

that way all the relational operators are implemented by exactly forwarding the operation to the `.value_` (a std::tuple).